tchannel-core/src/main/java/com/uber/tchannel/checksum/Checksums.java

```diff
@@ -21,25 +21,25 @@
  */
 package com.uber.tchannel.checksum;
 
-import com.uber.tchannel.messages.CallMessage;
+import com.uber.tchannel.frames.CallFrame;
```

Comment:
They are actually frame definitions. Naming them message is confusing. I understand to have message in the name is for the sake of netty channel/pipeline. However, that should be less focused in the world of TChannel.